packages/types/src/index.ts

```diff
@@ -574,6 +575,11 @@ export class CliUserConfigValidator extends SnippetShellUserConfigValidator {
           return `${key} must be a valid URL or empty`;
         }
         return null;
+      case 'logLocation':
+        if (value !== undefined && typeof value !== 'string') {
```

Comment:
I am wondering if there's a good way to check for valid path. I was thinking regex but worried it'd be end up disallowing valid paths in weirder OSes. Running `fs.access` could be a way, just seems to be a bit more proactive validator than others.